x-pack/plugins/monitoring/public/directives/beats/overview/index.js

```diff
@@ -21,9 +22,11 @@ uiModule.directive('monitoringBeatsOverview', (timefilter) => {
 
       function onBrush({ xaxis }) {
         scope.$evalAsync(() => {
```

Comment:
@nreese there are a bunch of these unnecessary `scope.$evalAsync()` calls, mind getting rid of them?